pkg/printers/humanreadable.go

```diff
@@ -95,26 +104,90 @@ func (h *HumanReadablePrinter) EnsurePrintHeaders() {
 // See validatePrintHandlerFunc for required method signature.
 func (h *HumanReadablePrinter) Handler(columns, columnsWithWide []string, printFunc interface{}) error {
 	printFuncValue := reflect.ValueOf(printFunc)
-	if err := h.validatePrintHandlerFunc(printFuncValue); err != nil {
-		glog.Errorf("Unable to add print handler: %v", err)
-		return err
+	entry := &handlerEntry{
+		columns:         columns,
+		columnsWithWide: columnsWithWide,
+	}
+	if err := ValidateRowPrintHandlerFunc(printFuncValue); err == nil {
```

Comment:
Just going top to bottom, so I haven't found callers, but given an if-block like this I would have expected two different functions instead of one which takes a "sometimes X or sometimes Y" argument.